### PR TITLE
Include sample request and response to dereference a remote item

### PR DESCRIPTION
**Pull request by chackman**
- Expand on example to dereference a remote item
- This is based on an answer to an existing GitHub issue [(#1000 in onedrive-api-docs)](https://github.com/OneDrive/onedrive-api-docs/issues/1000)

### CODE REVIEW

File: api-reference/beta/api/drive-sharedwithme.md [hunk 1]
```diff
@@ -91,22 +133,85 @@ Content-Type: application/json
...
+The following example shows how to access the shared **driveItem** that requires a request using the **id** of the **parentReference** within the **remoteItem** object.
+
+#### Request
+
+The following is an example of a request.
+
+<!-- {
+  "blockType": "request",
+  "name": "drives-get-remoteitem",
+  "sampleKeys": ["{remoteItem-parentReference-driveId}", "{remoteItem-id}"]
```
Comment (JarbasHorst) [inline api-reference/beta/api/drive-sharedwithme.md:165]: For clarity, please replace the placeholders, such as {remoteItem-parentReference-driveId}, with a real-looking ID. Examples with real-looking values in the, request URL, requests and response bodies help customers understand the API better. That's also the consistency bar for Microsoft Graph API docs.

Please, apply this change also here:
- line 170
- api-reference/v1.0/api/drive-sharedwithme.md

File: api-reference/beta/api/drive-sharedwithme.md [hunk 2]
```diff
@@ -91,22 +133,85 @@ Content-Type: application/json
...
+GET https://graph.microsoft.com/beta/drives/{remoteItem-parentReference-driveId}/items/{remoteItem-id}
+```
+
+#### Response
+
+The following is an example of the response.
+
+<!-- {
+  "blockType": "response",
+  "@odata.type": "",
```
Comment (JarbasHorst) [inline api-reference/beta/api/drive-sharedwithme.md:179]: Please, add the object type that this request returns. Would "microsoft.graph.driveItem" be correct?

Please, apply this change also here:
- api-reference/v1.0/api/drive-sharedwithme.md

File: api-reference/beta/api/drive-sharedwithme.md [hunk 3]
```diff
@@ -91,22 +133,85 @@ Content-Type: application/json
...
 ```
-By default, **sharedWithMe** returns items shared within your own tenant. To include items shared from external tenants, append `?allowexternal=true` to the GET request.
+
+#### Response
+
+The following is an example of the response.
+
+<!-- {
+  "blockType": "response",
+  "@odata.type": "",
```
Comment (JarbasHorst) [inline api-reference/beta/api/drive-sharedwithme.md:210]: Please, add the object type that this request returns. Would "microsoft.graph.driveItem" be correct?

Please, apply this change also here:
- api-reference/v1.0/api/drive-sharedwithme.md

File: api-reference/beta/api/drive-sharedwithme.md [hunk 4]
```diff
@@ -91,22 +133,85 @@ Content-Type: application/json
...
+  "truncated": true}
+-->
+
+
```
Comment (JarbasHorst) [inline api-reference/beta/api/drive-sharedwithme.md:183]: For consistency and clarity, please include the response object that this request returns. Something like:

```http
HTTP/1.1 200 OK
Content-Type: application/json

{
...
}
```

Please, apply this change also here:
- line 214
- api-reference/v1.0/api/drive-sharedwithme.md

**Comment by JarbasHorst** [comment]
Hello @chackman, I reviewed your PR, applied some changes to align the API topics with our standards, and left some comments. Please, take a look when you will have time.

File: api-reference/beta/api/drive-sharedwithme.md (hunk 1 above)
Comment (chackman) [reply api-reference/beta/api/drive-sharedwithme.md:165]: If we use real-looking values, then Examples 2 and 3 are redundant. I'll combine the two examples.

File: api-reference/beta/api/drive-sharedwithme.md (hunk 2 above)
Comment (chackman) [reply api-reference/beta/api/drive-sharedwithme.md:179]: Yes, it's a `driveItem`. I'll fix that.

File: api-reference/beta/api/drive-sharedwithme.md (hunk 4 above)
Comment (chackman) [reply api-reference/beta/api/drive-sharedwithme.md:183]: Will do

File: api-reference/beta/api/drive-sharedwithme.md (hunk 3 above)
Comment (chackman) [reply api-reference/beta/api/drive-sharedwithme.md:210]: This example will be combined with the previous example.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**